### PR TITLE
Accordion FED

### PR DESCRIPTION
**Pull request by liamjohnston**
Had to make some global CSS changes (e.g. font-size) to get it looking right. Also this is the first JS component, hence clearing up all the boilerplate JS setup while I was in there.

Re: The two-tone icons. This is what I was referring to. Can't just flip when open it as they're different icons.
Screenshot 2024-05-13 at 3 19 22 PM

Figs:
Screenshot 2024-05-13 at 3 39 26 PM

Browser:
Screenshot 2024-05-13 at 3 39 52 PM

### CODE REVIEW

File: cdhweb/static_src/components/ComponentsInit.tsx [hunk 1]
```diff
@@ -79,43 +77,6 @@ async function ComponentInit(
 
 const COMPONENT_ATTRIBUTE_NAME = 'data-component';
 
-// It's possible for components to appear inside other components. The most common example of this
```
Comment (liamjohnston) [inline cdhweb/static_src/components/ComponentsInit.tsx:82]: Not a thing for this project.

**Comment by cameralibre** [comment]
> can't just flip it when open

can't help but feel I'm being sniped, but...
https://codepen.io/cameralibre/pen/YzbPjzB

File: cdhweb/pages/templates/cdhpages/blocks/accordion_block.html [hunk 2]
```diff
@@ -1,17 +1,16 @@
 {% load wagtailcore_tags %}
 
+<div data-component="accordion" class="sk-accordion">
```
Comment (cameralibre) [inline cdhweb/pages/templates/cdhpages/blocks/accordion_block.html:3]: ooh, first time I've seen one of these `sk-` classes in the wild!

**Review by cameralibre — APPROVED** [review]
all looks good. though of course you can swap out the icons for the animated version above if it's not too much faff!

**Comment by liamjohnston** [comment]
That animation is extremely neat. However it'd mean loading those icons differently to all the others, so I'm going to skip it for now. Might be a NTH if I have tine at the end :)

**Pull request merged**